# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../xen-tools/compile.patch
# Copyright (C) 2009 The OpenSDE Project
# Copyright (C) 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- tools/xenstat/xentop/Makefile.orig	2009-08-02 18:15:18.000000000 +0300
+++ tools/xenstat/xentop/Makefile	2009-08-02 18:17:00.000000000 +0300
@@ -19,8 +19,8 @@
 else
 
 CFLAGS += -DGCC_PRINTF -Wall -Werror -I$(XEN_LIBXENSTAT)
-LDFLAGS += -L$(XEN_LIBXENSTAT)
-LDLIBS += -lxenstat $(CURSES_LIBS) $(SOCKET_LIBS)
+LDFLAGS += -L$(XEN_LIBXENSTAT) -L$(XEN_XENSTORE) -L../../libxc
+LDLIBS += -lxenstat $(CURSES_LIBS) $(SOCKET_LIBS) -lxenstore -lxenctrl
 CFLAGS += -DHOST_$(XEN_OS)
 
 .PHONY: all
